tests/unit_tests/scheduler/test_job.py

```diff
@@ -153,3 +158,101 @@ async def test_job_run_sends_expected_events(
         runpath=Path(realization.run_arg.runpath),
     )
     assert scheduler.driver.submit.call_count == max_submit
+
+
+@pytest.mark.asyncio
+async def test_job_run_logs_disk_sync_issue(realization: Realization, monkeypatch):
```

Comment:
The scenario in this test is not completely clear. Is it that the `_verify_checksum` times out? Maybe name it `test_when_waiting_for_disk_sync_times_out_an_error_is_logged`.